maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlFluentCommand.kt

```diff
@@ -61,27 +61,17 @@ data class YamlFluentCommand(
                 "scroll" -> MaestroCommand(scrollCommand = ScrollCommand())
                 else -> throw IllegalStateException("Unknown navigation target: $action")
             }
-            config != null -> MaestroCommand(
-                applyConfigurationCommand = ApplyConfigurationCommand(config),
-            )
-            else -> throw IllegalStateException("No mapping provided for $this")
+            else -> throw SyntaxError("Invalid command: No mapping provided for $this")
         }
     }
 
-    private fun launchApp(command: YamlLaunchAppUnion): MaestroCommand {
-        return when (command) {
-            is StringLaunchApp -> MaestroCommand(
```

Comment:
Having a `String` version would still be nice in case when user wants to open another app

Reply:
Updated